swarm/src/lib.rs

```diff
@@ -67,6 +67,25 @@ pub mod dummy;
 pub mod handler;
 pub mod keep_alive;
 
+/// Bundles all symbols required for the `libp2p_swarm_derive::NetworkBehavior` macro.
```

Comment:
```suggestion
/// Bundles all symbols required for the [`libp2p_swarm_derive::NetworkBehavior`] macro.
```

Reply:
This broke CI because it is not actually a link 🙃